### ISSUE TITLE
GeoWeightedSum ends with a python error

### ISSUE THREAD

**Issue by lluisvicens** [gmassei/VectorMCDA#3]
Every time I try to execute GeoWeightedSum algorithm in VectorMCDA, ends with an error:

```
An error has occurred while executing Python code: 

OSError: SI.out not found. 
Traceback (most recent call last):
  File "/Users/vicensboix/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/VectorMCDA/geoWeightedSum.py", line 319, in elaborate
    matrix=self.getAttributeMatrix()
  File "/Users/vicensboix/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/VectorMCDA/geoWeightedSum.py", line 405, in getAttributeMatrix
    self.pyMCDA(weight,preference)
  File "/Users/vicensboix/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/VectorMCDA/geoWeightedSum.py", line 409, in pyMCDA
    mat=support.inputFromTxt('SI.out')
  File "/Users/vicensboix/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/VectorMCDA/pymcda/support.py", line 42, in inputFromTxt
    SI = np.genfromtxt(path, dtype=None, delimiter=';', names=True)
  File "/Applications/QGIS3.10.app/Contents/MacOS/../Resources/python/numpy/lib/npyio.py", line 1744, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/Applications/QGIS3.10.app/Contents/MacOS/../Resources/python/numpy/lib/_datasource.py", line 266, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/Applications/QGIS3.10.app/Contents/MacOS/../Resources/python/numpy/lib/_datasource.py", line 624, in open
    raise IOError("%s not found." % path)
OSError: SI.out not found.


Python version: 3.7.3 (default, Mar 27 2019, 09:23:15) [Clang 10.0.1 (clang-1001.0.46.3)] 
QGIS version: 3.10.3-A Coruña A Coruña, 0e1f846438 

Python Path:
/Users/vicensboix/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/visualist
/Applications/QGIS3.10.app/Contents/MacOS/../Resources/python
/Users/vicensboix/Library/Application Support/QGIS/QGIS3/profiles/default/python
/Users/vicensboix/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins
/Applications/QGIS3.10.app/Contents/MacOS/../Resources/python/plugins
/Applications/QGIS3.10.app/Contents/Frameworks/Python.framework/Versions/Current/lib/python37.zip
/Applications/QGIS3.10.app/Contents/Frameworks/Python.framework/Versions/Current/lib/python3.7/site-packages
/Applications/QGIS3.10.app/Contents/Frameworks/Python.framework/Versions/Current/lib/python3.7/site-packages/geos
/Applications/QGIS3.10.app/Contents/Frameworks/Python.framework/Versions/Current/lib/python3.7
/Applications/QGIS3.10.app/Contents/Frameworks/Python.framework/Versions/Current/lib/python3.7/lib-dynload
/Applications/QGIS3.10.app/Contents/Resources/python
/Users/vicensboix/Library/Application Support/QGIS/QGIS3/profiles/default/python
```
I tried this algorithm in QGIS 3.10 on Mac (10.15) and Windows with the same result and message. On the other hand, GeoTOPSYS works pretty well.
Thanks in advance!

**Comment by gmassei** [comment]
The bug should be solved with vectorMCDA 3.0.1 version.
Hy
GM

**Comment by lluisvicens** [comment]
> The bug should be solved with vectorMCDA 3.0.1 version.
> Hy
> GM

I can confirm this. It works fine again. Thanks!
Cheers,
Lluís
